tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -23,15 +23,18 @@ def parse_datafile(datafile):
         util.make_generate_parser_error(datafile, 1)
     )
     if not header_is_valid:
+        logger.info(f"Preparser Error -> Header errors: {[error for error in header_errors]}")
```

Comment:
I believe Jan made a comment on this as well -- can we just get a count here vs listing them all out explicitly? This is at info level so count feels more correct whereas debug we could list everything but I think we do a full dump elsewhere with parsing.